_posts/2021-02-11-highs.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "HiGHS.jl 0.1 released"
+date:   2021-02-11
+categories: [solvers]
+---
+
+We are happy to announce the initial `v0.1` release of [HiGHS.jl](https://github.com/jump-dev/HiGHS.jl),
+a wrapper for [HiGHS](https://www.maths.ed.ac.uk/hall/HiGHS/).
+
+HiGHS is a new high-performance open-source linear programming solver being
+developed by Julian Hall and colleagues at the University of Edinburgh,
+Scotland.
+
+You can try it out as follows:
+```julia
+import Pkg; Pkg.add("HiGHS")
+
+using JuMP, HiGHS
+model = Model(HiGHS.Optimizer)
+set_silent(model)
```

Comment:
Why is `set_silent` needed in the example?

Reply:
HiGHS is very talky, and `set_optimizer_attribute(model, "message_level", 0)` still prints a lot.

We could change it to `# set_silent(model)  # Uncomment to turn off printing`?

Reply:
We can leave as is, was mostly curious.

Reply:
We've also discussed how, in due course, HiGHS will be stopped from chatting in a clearer and simpler manner